core/common/src/main/java/alluxio/network/protocol/RPCBlockReadResponse.java

```diff
@@ -24,6 +24,8 @@
 
 /**
  * This represents the response of a {@link RPCBlockReadRequest}.
+ *
+ * Since 1.4.1, this also represents the response of a {@link RPCUnderFileSystemBlockReadRequest}.
```

Comment:
and I do not think we should be releasing your changes as part of a patch release